pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -187,17 +187,20 @@ def __init__(
 
         self.kth_value, self.mode = mode_dict[mode]
 
+    @rank_zero_only
     def _del_model(self, filepath: str):
         if self._fs.exists(filepath):
             self._fs.rm(filepath)
+            log.info(f"Removed checkpoint: {filepath}")
```

Comment:
```suggestion
            log.debug(f"Removed checkpoint: {filepath}")
```
I would lower nb messages...